charts/dify/values.yaml

```diff
@@ -302,6 +313,17 @@ postgresql:
         - ReadWriteOnce
       size: 8Gi
 
+    ## @param readReplicas.affinity Affinity for PostgreSQL read only pods assignment
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    ## Note: primary.podAffinityPreset, primary.podAntiAffinityPreset, and primary.nodeAffinityPreset will be ignored when it's set
+    ##
+    affinity: {}
+
+    ## @param primary.tolerations Tolerations for PostgreSQL primary pods assignment
```

Comment:
Comment here suggests `primary` but indeed they are for `readReplicas`. Please update them as follow:
```
  tolerations: []
  ## @param readReplicas.topologySpreadConstraints Topology Spread Constraints for pod assignment spread across your cluster among failure-domains. Evaluated as a template
  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/#spread-constraints-for-pods
  ##
```